### PR TITLE
feat: testing in go 1.10 version.

### PR DESCRIPTION
**Pull request by appleboy**
Testing with 1.10 version using Golang official image.

### CODE REVIEW

**Comment by thehowl** [comment]
Why were we using the webhippie image in the first place? cc/ @tboerger - does it have any advantages over the standard golang image?

Integration tests seem to fail because we don't have git lfs - might need to install it on the image

**Comment by appleboy** [comment]
@thehowl I already add `git-lfs` package. Waiting drone buid.

**Comment by appleboy** [comment]
@thehowl 

> Why were we using the webhippie image in the first place? cc/ @tboerger - does it have any advantages over the standard golang image?

So we move to the standard golang image now because of webhippie is not the latest golang image.

**Comment by tboerger** [comment]
I can add the latest version... The official go image doesn't include make ;)

**Comment by appleboy** [comment]
@tboerger official image support `make` command. See:

https://github.com/docker-library/golang/blob/329b6672949337c4469351b2e8b8575459a75c7f/1.10/stretch/Dockerfile#L8

just missing `git-lfs` package which already fixes in this PR.

**Comment by lunny** [comment]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -56,21 +56,18 @@ pipeline:
       event: [ push, tag, pull_request ]
 
   build-without-gcc:
-    image: webhippie/golang:1.8
+    image: golang:1.10
     pull: true
```
Comment (lafriks) [inline .drone.yml]: This should be left as golang:1.8

**Review by lafriks — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (appleboy) [reply .drone.yml]: https://github.com/go-gitea/gitea/pull/3899/commits/e054e92104098c61caf82172592b0fbf5dd06348 Done!

**Review by lafriks — APPROVED** [review]

**Pull request merged**